packages/editor/src/client/login.tsx

```diff
@@ -193,7 +225,10 @@ export function Login() {
                 type="password"
                 value={password}
                 autoComplete={'currentPassword'}
-                onChange={password => setPassword(password)}
+                onChange={(password: string) => setPassword(password)}
+                inputRef={(ref: HTMLInputElement): void => {
```

Comment:
Instead of `inputRef` i suggest to use `onKeyDown`. Makes the whole thing a bit simpler and you don't require a document event listener.

Reply:
It's good suggestion 👍 I try it before I implement this using ref but doesn't work 😄 now it's good 🤣 I change this

Reply:
Sorry for so many commits but I forget when i change to onKeyDown to leave refs, because they are needed to checking focus.

Reply:
Hey @treflxyz,

Thanks for the contribution to wepublish. I took over Github project maintenance from Nico so I'll be responsible for accepting external changes. I agree with Nico that we should make it as simple as possible with the simple `onKeyDown` per element. 

Or can you explain why you reverted back to refs?
Why you need to check focus? `onKeyDown` triggers event only when actually pressed on that certain element.

Reply:
Hey @treflxyz,

Thanks for the contribution to wepublish. I took over Github project maintenance from Nico so I'll be responsible for accepting external changes. I agree with Nico that we should make it as simple as possible with the simple `onKeyDown` per element. 

Or can you explain why you reverted back to refs?
Why you need to check focus? `onKeyDown` triggers event only when actually pressed on that certain element.